src/index.html

```diff
@@ -4,24 +4,410 @@
+            <a
+              class="device_category nav__link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="device_category nav__link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="device_category nav__link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="device_category nav__link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="  APPLE A1419 iMac"
+          />
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__cost">
+            <p class="card__price">Price:</p>
+            <p class="card__total">$2,199</p>
+          </div>
+
+          <button
+            data-qa="card-hover"
+            class="card__button"
+          >
+            Buy
+          </button>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="  APPLE A1419 iMac"
+          />
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__cost">
+            <p class="card__price">Price:</p>
+            <p class="card__total">$2,199</p>
+          </div>
+
+          <button
+            data-qa="hover"
```

Comment:
The `data-qa` attribute value is inconsistent with other similar elements. Consider using a consistent naming convention for `data-qa` attributes to maintain uniformity across similar elements.